stream/distributedlog/core/src/main/java/org/apache/distributedlog/BKAsyncLogWriter.java

```diff
@@ -465,9 +465,24 @@ CompletableFuture<Long> flushAndCommit() {
         if (null == writerFuture) {
             return FutureUtils.value(getLastTxId());
         }
-        return writerFuture.thenCompose(writer -> writer.flushAndCommit());
+        return writerFuture.thenCompose(BKLogSegmentWriter::flushAndCommit);
```

Comment:
unrelated and unnecessary change... please keep as it is

Reply:
reverted the change